src/components/options/settings/main.svelte

```diff
@@ -29,7 +27,7 @@ function mainConfigurationUpdated(lazy) {
   if (!lazy) {
     updateLastConfigUpdate();
   }
-  dispatch("updateMainConfiguration", lazy);
+  chrome.runtime.sendMessage({ action: "updateContextualMenu" });
```

Comment:
With this change, you are doing the equivalent to _always_ doing `dispatch("updateMainConfiguration", true);`, **however** when the `updateNow` function is invoced, `lazy` will be False, hence I have the _feeling_ (I haven't tried yet) that the UI component for providers isn't being updated when the users clicks the "Update now" button.